### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by TheEpikPikmin**
I added the images without having to put them in the folder itself.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,28 @@
...
+## Updates
+### Design
+![Design](https://github.com/samjbeef/EstiGuesser/assets/73117832/60a1db68-a119-44a4-a50e-cbaf5a144a0c)
+
+### Code Explained
+![Code_3](https://github.com/samjbeef/EstiGuesser/assets/73117832/d6394aac-00b6-4594-86a9-38b95c1e768a)
+![Code_2](https://github.com/samjbeef/EstiGuesser/assets/73117832/1d9547e4-c6eb-4e2e-964d-5f121b26e723)
+![Code_1](https://github.com/samjbeef/EstiGuesser/assets/73117832/a868bb43-aaed-45d6-adf3-543a87703963)
+
+## Milestones
```
Comment (fkamau1) [inline README.md]: We can take this one out

File: README.md [hunk 2]
```diff
@@ -1 +1,28 @@
 # EtsiGuesser
+## The real estate price guessing game
+
+## Overview
+EstiGuessr is a browser-based game where you, the player, will be provided with pictures of homes for sale through the MLS or similar API. You will then have to use that, combined with the location of the home to guess it’s price after which you will be scored based on how close your guess was. Once on the website you will be prompted with a username, then your best score will then be added to a leaderboard where you can try to reach the top 10 scores. The website will be hosted via Amazon Web Services.
+
+## About Us
```
Comment (fkamau1) [inline README.md]: We can take this out as well

**Review by fkamau1 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (fkamau1) [reply README.md]: I messed this one it should just be Milestone

**Review by samjbeef — APPROVED** [review]

**Review by samjbeef — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (fkamau1) [reply README.md]: Same here it should just be take out about us

**Pull request merged**